File: Assets/AltUnityTester/AltUnityDriver/Commands/OldFindObjects/FindElementByComponent.cs
public class FindElementByComponent : CommandReturningAltElement
{
    string componentName;
    string assemblyName;
    string cameraName;
    bool enabled;
    public FindElementByComponent(SocketSettings socketSettings, string componentName, string assemblyName, string cameraName, bool enabled) : base(socketSettings)
    {
        this.componentName = componentName;
        this.assemblyName = assemblyName;
        this.cameraName = cameraName;
        this.enabled = enabled;
    }
    public AltUnityObject Execute(){
        Socket.Client.Send(toBytes(CreateCommand("findObjectByComponent", assemblyName, componentName, cameraName, enabled.ToString())));
        return ReceiveAltUnityObject();
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/OldFindObjects/find_element_by_component.py
from altunityrunner.commands.command_returning_alt_elements import CommandReturningAltElements
class FindElementByComponent(CommandReturningAltElements):
    def __init__(self, socket,requestSeparator,requestEnd,component_name,assembly_name,camera_name,enabled):
        super().__init__(socket,requestSeparator,requestEnd)
        self.component_name=component_name
        self.assembly_name=assembly_name
        self.camera_name=camera_name
        self.enabled=enabled
    
    def execute(self):
        if self.enabled==True:
            data = self.send_data(self.create_command('findObjectByComponent',self.assembly_name, self.component_name , self.camera_name ,'true'))
        else:
            data = self.send_data(self.create_command('findObjectByComponent',self.assembly_name, self.component_name , self.camera_name ,'false'))
        return self.get_alt_element(data)